modules/table/src/main/java/org/apache/ignite/internal/table/distributed/raft/snapshot/PartitionAccessImpl.java

```diff
@@ -192,6 +192,8 @@ public long maxLastAppliedTerm() {
 
     @Override
     public CompletableFuture<Void> startRebalance() {
+        storageUpdateHandler.waitIndexes();
```

Comment:
Please add a clarifying comment, why this line is here

Reply:
Fix it.